client/src/components/Profile/Tests/Profile.test.js

```diff
@@ -24,6 +24,6 @@ describe("<Profile>", () => {
 
   test("get user information", () => {
     const wrapper = shallow(<Profile {...props} />);
-    expect(props.getUser).toHaveBeenCalled();
+    // expect(props.getUser).toHaveBeenCalled();
```

Comment:
we should remove or change this line at some point if we're going to keep it commented

Reply:
Sounds good, I removed it as we are not calling getUsers anymore